docs/products/clickhouse/index.rst

```diff
@@ -0,0 +1,40 @@
+ClickHouse :badge:`beta,cls=badge-secondary text-black badge-pill`
+==================================================================
+
+What is Aiven for ClickHouse?
+-----------------------------
+
+Aiven for ClickHouse beta is powered by ClickHouse, an open-source database management system that uses a column-oriented structure and is best suited for the online analytical processing of queries (OLAP).
```

Comment:
one more thing which we can add is that, especially in case of clickhouse, which is a truly columnar database,  the values from the same column will be physically stored next to each other. That's why retrieving a column is so fast. On the other hand, the values belonging to the same raw are stored across different physical locations, so retrieving complete item rows is slow.

Reply:
> Really enjoyed reading this!
> 
> I added couple of comments and suggestions. Also, I wonder if "Supported table engines" really belongs to concepts. But it doesn't feel into HowTos as well.

Thanks for the comments - the content is now updated based on those. And I agree, the concepts section might not be the best place for the "Supported table engines" article, so I've moved that under "References" ;)